pkg/resource/configmap/list_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Kubernetes Authors.
```

Comment:
Is this file copied from Kubernetes?

Reply:
yes, should I follow the rules of lift: https://karmada.io/docs/contributor/lifted

Reply:
Not really. It looks unnormal to just copy test files. 
Don't have to be that complex.
But can you add a comment right after the license to explain the source link?

Reply:
Got it. After reviewing that test file again, it's no need to lift that file from `kubernetes/dashboard` repo, so I decided deleting that test file.